docs/operations.md

```diff
@@ -123,6 +171,128 @@ stays in the database:
 
 **Default**: `30` (days)
 
+### Scavenge database option
```

Comment:
there are already database options described above this section, would probably sensible to merge them together. it's a bit odd at the moment 
![image](https://github.com/EventStore/EventStore/assets/2587665/1684e4db-e911-4bf0-a47a-f71d549d25e5)

Reply:
Thank you I fixed